k8s/demo/common-overlay/am/kustomization.yaml

```diff
@@ -3,3 +3,6 @@ kind: Kustomization
 namespace: am
 bases:
 - ../../../namespaces/am
+- ../../../namespaces/am/am-role-assignment-service/am-role-assignment-service.yaml
```

Comment:
I don't think this is quite right, you've added it to the global kustomization and the demo one? don't you only need to add it to one?

Reply:
Global as in k8s/namespaces/am/kustomization.yaml ? I have not added here. I want this service to be deployed only on AAT and Demo for now.